### PR TITLE
sci-electronics/librepcb: up to 0.1.6

### PR DESCRIPTION
**Pull request by ktrace**
- switch from qmake to cmake
- unbundle dev-cpp/muParser and dev-libs/quazip
- backport librepcb-0.1.6-muparser-fix.patch from master
- bugreport QA warning to upstream

Package-Manager: Portage-3.0.30, Repoman-3.0.3

### CODE REVIEW

File: sci-electronics/librepcb/librepcb-0.1.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+src_configure() {
+	local mycmakeargs+=(
+	    -DUNBUNDLE_QUAZIP=ON
+	    -DUNBUNDLE_MUPARSER=ON )
+	cmake_src_configure
+}
+
+src_test() {
+	cd "${BUILD_DIR}"/tests/unittests
```
Comment (thesamesam) [inline sci-electronics/librepcb/librepcb-0.1.6.ebuild:48]: missing || die on both

**Comment by ktrace** [comment]
sry, forgot patch https://github.com/LibrePCB/LibrePCB/commit/b65dc1b766fc54da3117d44f612f6a93d5caa602

File: sci-electronics/librepcb/librepcb-0.1.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+src_configure() {
+	local mycmakeargs+=(
+	    -DUNBUNDLE_QUAZIP=ON
+	    -DUNBUNDLE_MUPARSER=ON )
+	cmake_src_configure
+}
+
+src_test() {
+	cd "${BUILD_DIR}"/tests/unittests || die
+	virtx ./librepcb-unittests --gtest_filter=-SystemInfoTest.testGetUsername || die
```
Comment (juippis) [inline sci-electronics/librepcb/librepcb-0.1.6.ebuild:49]: `virtx` and `|| die` should only be used together when the return code is known.

**Review by juippis — APPROVED** [review]
Tests fail, but otherwise seems to be working fine now. Thanks!

**Pull request closed without merging**